app/models/publication.rb

```diff
@@ -86,8 +86,7 @@ def self.build_new_manual_publication(pub_hash, original_source_string, provenan
 
   # @return [self]
   def update_manual_pub_from_pub_hash(incoming_pub_hash, original_source_string, provenance = Settings.cap_provenance)
-    incoming_pub_hash[:provenance] = provenance
-    self.pub_hash = incoming_pub_hash.dup
+    self.pub_hash = incoming_pub_hash.merge(provenance: provenance)
     match = UserSubmittedSourceRecord.find_by_source_data(original_source_string)
```

Comment:
:+1:

Reply:
299add1be9a4fc84a100103fcad14e547468fc13 approved - requesting it is squashed with prior commits that change specs by removing this parameter there also - noting that this PR review would have been easier if they were merged and consolidated into one PR (less guess work during the review)

Reply:
Noting that this change means our API cannot have additional manual consumers outside the 'cap' consumer after this change is made.  In other words, this change ensures there can only ever be one source of manual publications.  That it, it entails, for example, that there could be other Stanford (or other) consumers of this API that we would want to identify as a source (provenance) for manual submissions.  As we have no use case for that now, this might be OK; but if we do discover such a use case, this change might need to be reverted.  Hence, consider crafting the commit/PR for just removing this method parameter so it could be easily reverted.

Reply:
I'm trying to think of whether we have any additional use cases where we need to keep the method param - I suppose we consider all bibtex ingests as belonging to a cap provenance.  So maybe not and the git grep in the commit message seems to suggest not.

Reply:
Can't get much tighter than 4 lines touched.